kwok/apis/crds/karpenter.kwok.sh_kwoknodeclasses.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.16.3
+    controller-gen.kubebuilder.io/version: v0.16.4
```

Comment:
Can we (you) split this bump into its own commit / explain more about the context?

Reply:
This bump was created by running `make verify` and then presumably by `go generate ./...` - as this is my first time working on karpenter I wasn't sure if I should commit this change. Happy to remove the bumps if it is deemed not necessary.

Reply:
(I would make the autogenerated changes a separate commit, then it is easy to omit if appropriate)